websites/K/Keep Talking and Nobody Explodes/presence.ts

```diff
@@ -2,158 +2,215 @@ const presence = new Presence({
     clientId: "681116862930747520"
   }),
   strings = presence.getStrings({
-    browsing: "presence.activity.browsing",
-    reading: "presence.activity.reading"
+    reading: "presence.activity.reading",
+    page: "general.page"
   });
 
-let browsingTimestamp = 0; //Last started activity
+let timestamp = 0,
+  previous: Location,
+  current: Location;
 
 presence.on("UpdateData", async () => {
-  const path = window.location.pathname.split("/").slice(1),
+  current = window.location;
+  const path = current.pathname.split("/").slice(1);
+
+  if (current.hostname.split(".")[0] === "bombmanual") {
+    // Bomb manual page
+    switch (isLanguageCode(path[0]) ? path[1] : path[0]) {
+      // Web manual
+      // PDF manual
+      case "web":
+      case "print":
+        await handleManual(isNewLocation(previous, current));
+        break;
+      // How to play
+      // Language select
+      case "how-to-play-pc.html":
+      case "how-to-play-mobile.html":
+      case "how-to-play-switch.html":
+      case "how-to-play-xbox.html":
+      case "how-to-play-playstation.html":
+      case "how-to-play-vr.html":
+      case "how-to-play-psvr.html":
+      case "how-to-play-gear-vr.html":
+      case "how-to-play-oculus-go.html":
+      case "how-to-play-oculus-quest.html":
+      case "how-to-play-daydream.html":
+      case "language":
+        await handleGeneric(true);
+        break;
+      // Startpage
+      // Unknown
+      default:
+        presence.setActivity();
+        break;
+    }
+  } else {
+    // Main page
+    switch (path[0]) {
+      // Contact Us
+      // Privacy Policy
+      case "contact-us":
+      case "privacy-policy":
+        await handleGeneric();
+        break;
+      // Presskit
+      case "presskit":
+        await handleCustom("Presskit");
+        break;
+      // How To Play Remotely
+      case "how-to-play-remotely":
+        await handleCustom("How To Play Remotely");
+        break;
+      // Mobile app
+      case "mobile":
+        await handleCustom("Mobile App");
+        break;
+      // Translation FAQ
+      case "translation-faq":
+        await handleCustom("Translation FAQ");
+        break;
+      case "faq":
+        await handleFAQ();
+        break;
+      // Commercial Licensing
+      case "commercial-license":
+        await handleCustom("Commercial Licensing");
+        break;
+      // Non-Commercial Use
+      case "non-commercial-use":
+        await handleCustom("Non-Commercial Use");
+        break;
+      // Startpage
+      // Unknown
+      default:
+        presence.setActivity();
+        break;
+    }
+  }
+
+  previous = current;
+});
+
+/**
+ * Handle bomb manual
```

Comment:
What

Reply:
What exactly do you mean? This part is responsible for the bomb manual that's required to play the game. You can view it on this page for example: https://bombmanual.com/web/index.html